ui/tests/acceptance/oidc-auth-method-test.js

```diff
@@ -52,7 +52,7 @@ module('Acceptance | oidc auth method', function (hooks) {
   });
 
   test('it should login with oidc from listed auth mount tab', async function (assert) {
-    assert.expect(2);
+    assert.expect(3); // request is fired more than once -- so three assertions instead of two
```

Comment:
Can we add a TODO to update the actual behavior, if it's firing more than expected?

Reply:
I think this is from `fetchRole` being triggered in the `didReceiveAttrs` hook of the `auth-jwt` component. When the component is instantiated it fetches the role and then again when the auth method tab is clicked (triggers on re-render). `didUpdateAttrs` is probably better to use in this case but then again there may be a better approach when the component is glimmerized. I'll add a comment to the test to capture this.